bin/enable-executable

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env ruby
 puts 'Ensuring test files are executable...'
 
-Dir.glob('**/*test.rb').each do |f|
+executable_files = Dir.glob('**/*test.rb') + Dir.glob('bin/*')
+executable_files.each do |f|
   if File.executable?(f)
     print '.'
   else
     print "Adding exec bit to #{f}."
-    File.chmod(0744, f)
+    File.chmod(0775, f)
```

Comment:
Why the expanded privileges here? 744 should be fine, and technically more permissive even than necessary.